### ISSUE TITLE
A11y: Footer: Move decorative dash span item inside the list item, and hide them from screenreaders

### ISSUE THREAD

**Issue by bacitracin** [CodeForPhilly/clean-and-green-philly#856]
## Describe the task

<img width="692" alt="Screenshot 2024-08-02 at 11 00 28 AM" src="https://github.com/user-attachments/assets/1add6d56-b072-4605-b288-06057a83aff1">

There are a couple of issues with the Footer.

1. The footer nav is an unordered list. Unordered lists should only contain list items. The decorative dashes are inside span elements. The spans need to be moved inside the list items.
![Screenshot 2024-08-02 at 11 06 08 AM](https://github.com/user-attachments/assets/7bdb728d-9afd-4886-acdb-2cb7441e3208)

2. The decorative dashes are announced when navigating through the footer with a screenreader. Add an `aria-hidden="true"` to these spans so they're not announced.

## Acceptance Criteria

- [ ] When navigating through the footer content with a screenreader turned on, the dashes are not landed on/announced.
- [ ] The dashes still appear  in desktop mode, and disappear in mobile view.

**Comment by zheyichn** [comment]
Hi, can I be assigned this issue? Thanks.

**Comment by CodeWritingCow** [comment]
@zheyichn assigned!
